### PR TITLE
add: future delay

### LINKED ISSUE #107: Create account flow broken on iOS
## Generic information
- OS name and version: iOS
- Application build code: `4200` 
- Network type (mobile/Wi-Fi): Both
- Reporting user: @lauranori 

## Bug description
Inside the login screen, after tapping the "Create account" button using an iOS device it is not possible to come back to the login page. 
This makes it very frustrating for the users who then have to close and re-open the app due to a missing back button.

## Steps to reproduce
1. Open the login screen
2. Tap the "Create a new account" button

## Expected behavior
The user should be able to come back to the login screen after tapping that button. This is to prevent any frustrating app reopening after tapping on such button inadvertently.

### LINKED ISSUE #108: Create account too immediate
## Generic information
- OS name and version: Android/iOS
- Application build code: `4200`
- Network type (mobile/Wi-Fi): Both
- Reporting user: @lauranori 

## Bug description
Inside the login screen, the creation flow is too fast. When the user taps on the "Create account" button, the switch to the new page is too fast. This can lead to an unwanted sense of discomfort due to the fast reaction of the app. 

## Steps to reproduce
1. Go inside the login page
2. Tap on the "Create account" page

## Expected behavior
A solution to solve this might be to put a warning popup telling the user what that function does and how it will behave. This will create a new small time gap between the login screen and the account creation one.

### PR DESCRIPTION
**Pull request by ryuash**
## Description

closes #108 closes #107

Screen Shot 2020-08-18 at 3 17 18 PM

## Checklist
- [x] Targeted PR against correct branch.
- [x] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [x] Wrote unit tests.
- [x] Run flutter format.
- [ ] Updated the documentation.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

File: lib/ui/blocs/account_read/account_bloc.dart [hunk 1]
```diff
@@ -117,6 +117,7 @@ class AccountBloc extends Bloc<AccountEvent, AccountState> {
   Stream<AccountState> _mapGenerateAccountEventToState() async* {
     yield CreatingAccount();
     final mnemonic = await _generateMnemonicUseCase.generate();
+    await Future.delayed(const Duration(seconds: 3));
```
Comment (rmontagnin) [inline lib/ui/blocs/account_read/account_bloc.dart]: Can we change this to be 2 seconds? I think 3 might look a little bit too long

```suggestion
    await Future.delayed(const Duration(seconds: 2));
```

**Review by rmontagnin — COMMENTED** [review]
I think it might be better to also add a popup that is displayed when the user clicks on the "Create account" button telling what that does. This will allow the user to go back if he presses it by error. What do you think? 

Also, could you add a CHANGELOG entry?

**Review by rmontagnin — APPROVED** [review]

**Comment by rmontagnin** [comment]
@ryuash Looks like a test is failing. Can you fix that please?

**Comment by ryuash** [comment]
fixed

**Pull request merged**